File: 1038.c
#include <stdio.h>
int main(){
    int b;
    double a;
    scanf("%lf%i", &a, &b);
    if(a == 1.0){
        a = 4.0;}
    else if(a == 2.0){
        a = 4.5;}
    else if(a == 3.0){
        a = 5.0;}
    else if(a == 4.0){
        a = 2.0;}
    else if(a == 5.0){
        a = 1.5;}
    printf("Total: R$ %.2lf\n", a*b);
}

File: 1038.cs
using System;
using System.Globalization;
class URI {
    static void Main(string[] args) { 
        String [] x = Console.ReadLine().Split(' ');
        double a;
        int b = int.Parse(x[1]);
        a = 0;
        if(x[0] == "1"){
            a = 4.0;
        }
        if(x[0] == "2"){
            a = 4.5;
        }
        if(x[0] == "3"){
            a = 5.0;
        }
        if(x[0] == "4"){
            a = 2.0;
        }
        if(x[0] == "5"){
            a = 1.5;
        }
        Console.WriteLine($"Total: R$ {a*b:F2}");
    }
}
